pt-BR/apple_description_release.lang

```diff
@@ -46,11 +46,11 @@ Spend your time reading your favorites sites instead of looking for them.
 
 # Slogan in uppercase, displayed as a category title
 ;SYNC FIREFOX ACROSS YOUR DEVICES
-SYNC FIREFOX ACROSS YOUR DEVICES
+TODOS SEUS DEVICES SINCRONIZADOS COM FIREFOX SYNC
 
 
 ;With a Firefox Account, access your history and open tabs everywhere you use Firefox.
-With a Firefox Account, access your history and open tabs everywhere you use Firefox.
+Com sua Conta Firefox, acesse seu histórico e abas abertas em qualquer lugar que usar Firefox.
```

Comment:
You guys are right, of course. I got this confused with "Firefox Sync". You can merge with the string like this.